### ISSUE TITLE
'undefined' is not an object (evaluating '$(location.hash.trim()).offset().top')

### ISSUE THREAD

**Issue by yegor256** [yegor256/netbout#731]
I keep getting this error (5-10 times per day) from qbaka:

```
'undefined' is not an object (evaluating '$(location.hash.trim()).offset().top')

Error location:	http://localhost:18727/js/bout.js at line 5
User location:	http://localhost/b/1435598696141
Happened at:	2015.06.29 19:24:57
```

**Comment by yegor256** [comment]
@dmzaytsev looks like this happens on local machines only, when someone is testing netbout or just running it locally

**Comment by dmzaytsev** [comment]
@yegor256 I think if all of the errors from localhost we can close this ticket,
otherwise we will investigate this case
what do you think?

p.s. the port value 18727 means that is test case

**Comment by yegor256** [comment]
@dmzaytsev why closing? we shouldn't fail even in tests... let's fix it

**Comment by dmzaytsev** [comment]
@yegor256 OK, will do

**Comment by dmzaytsev** [comment]
@karato it's a valid bug

**Comment by karato** [comment]
> @karato it's a valid bug

@dmzaytsev I just added bug tag here

**Comment by karato** [comment]
@yegor256 since the ticket has no milestone I set it to 3.1

**Comment by karato** [comment]
@yegor256 thanks for the report, I topped your acc for 15 mins, payment ID `000-95c9aa70`

**Comment by antonini** [comment]
Issue #728 is going to fix it.

**Comment by antonini** [comment]
@yegor256 Check snapshot. The error is gone.

**Comment by dmzaytsev** [comment]
@yegor256 could you check the Qbaka log after `2015-07-10T03:39:47 GMT`? 
if you will see a message like `'undefined' is not an object (evaluating '$(location.hash.trim()).offset().top')` 
let me know please

**Comment by yegor256** [comment]
@antonini I see that #728 is still open... in which pull request this problem was fixed?

**Comment by antonini** [comment]
@yegor256 It was fixed on PR #735.

**Comment by yegor256** [comment]
@antonini ok, thanks!
